content/en/mobile/datadog_for_intune.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Datadog for Intune
+further_reading:
+- link: "https://www.datadoghq.com/blog/datadog-mobile-widgets/"
+  tag: "Blog"
+  text: "Improve your on-call experience with Datadog mobile dashboard widgets"
+---
+
+This guide provides step-by-step instructions to configure and deploy the Datadog for Intune mobile app within your organization.
+
+## Prerequisites
+Before you begin, ensure the following requirements are met:
+
+- Admin permissions in **Intune, Azure, and Datadog**
+- Users must download and install the **Datadog for Intune** app from their mobile app store or Microsoft Partner store.
+
+For those looking to create a custom configuration using the mobile app bundle ID, refer to the links below:
+
+| Platform   | Store Link                                                             |
+|------------|------------------------------------------------------------------------|
+| iOS/iPadOS | [Datadog Intune on the App Store](https://apps.apple.com/app/datadog-intune/id1673106321) |
+| Android    | [Datadog Intune on Google Play](https://play.google.com/store/apps/details?id=com.datadog.app.intune) |
```

Comment:
`BundleID` are necessary here since sometimes Microsoft asks for it

Reply:
Fixed